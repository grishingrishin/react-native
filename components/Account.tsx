import React from 'react';
import styled from 'styled-components/native';
import Actions from './Actions';

const Container = styled.View`
  flex: 1;
  background-color: pink;
`;

export default () => (
  <Container>
    <Actions />
  </Container>
);
