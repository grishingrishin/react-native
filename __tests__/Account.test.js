import React from 'react';
// import {shallow} from 'enzyme';
import {create} from 'react-test-renderer';
import Account from '../components/Account';

describe('<Account />', () => {
  describe('Jest tests', () => {
    it('renders correctly', () => {
      const tree = create(<Account />).toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});