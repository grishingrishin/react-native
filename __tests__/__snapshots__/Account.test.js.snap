// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Account /> Jest tests renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "pink",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
      },
    ]
  }
>
  <View
    style={
      Array [
        Object {
          "alignItems": "center",
          "flexDirection": "row",
          "justifyContent": "space-between",
          "left": 0,
          "paddingBottom": 28,
          "paddingLeft": 24,
          "paddingRight": 24,
          "paddingTop": 28,
          "position": "absolute",
          "top": 0,
          "width": "100%",
          "zIndex": 1,
        },
      ]
    }
  >
    <View
      style={
        Array [
          Object {
            "marginRight": 24,
          },
        ]
      }
    >
      <View
        accessible={true}
        focusable={false}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "opacity": 1,
          }
        }
      >
        <Image
          source={
            Object {
              "testUri": "../../../assets/hamburger.png",
            }
          }
          style={
            Array [
              Object {
                "height": 24,
                "width": 42,
              },
            ]
          }
        />
      </View>
    </View>
    <View
      style={
        Array [
          Object {
            "alignItems": "center",
            "flexDirection": "row",
            "justifyContent": "center",
          },
        ]
      }
    >
      <View
        style={
          Array [
            Object {
              "marginLeft": 24,
            },
          ]
        }
      >
        <View
          accessible={true}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "opacity": 1,
            }
          }
        >
          <View
            style={
              Array [
                Object {
                  "alignItems": "center",
                  "flexDirection": "row",
                },
              ]
            }
          >
            <Image
              source={
                Object {
                  "testUri": "../../../assets/search.png",
                }
              }
              style={
                Array [
                  Object {
                    "height": 32,
                    "marginLeft": 8,
                    "width": 32,
                  },
                ]
              }
            />
          </View>
        </View>
      </View>
      <View
        style={
          Array [
            Object {
              "marginLeft": 24,
            },
          ]
        }
      >
        <Image
          source={
            Object {
              "testUri": "../../../assets/user.png",
            }
          }
          style={
            Array [
              Object {
                "height": 48,
                "width": 48,
              },
            ]
          }
        />
      </View>
    </View>
  </View>
</View>
`;
